models/modelArticles.js

```diff
@@ -17,7 +17,12 @@ exports.fetchArticlesById = (article_id) => {
     if (!inc_votes)
       return Promise.reject({ status: 400, msg: "Bad request: Missing input." });
     return connection
-    .query('UPDATE articles SET votes = $1 WHERE article_id =$2 RETURNING*', [inc_votes,article_id]).then(({rows}) =>{
-        return rows[0]
+    .query('UPDATE articles SET votes = $1 WHERE article_id =$2 RETURNING*', [inc_votes,article_id]).then(({rows}) => {

```

Comment:
is this change from a previous PR if you need to make changes you should switch branches and then commit and re-merge from that branch, this will avoid this kind of stuff :)